plugin/js/import.js

```diff
@@ -71,38 +71,43 @@ function processNextItem(firstOrLast, buildAjaxCallback, callerButton, url_base,
     let itemType      = elementToProcess.data('itemtype');
     let ajaxUrl       = url_base + '/ajax/' + buildAjaxCallback(itemType, startFromId);
     resultElement.addClass('fa-spinner fa-pulse');
-    resultElement.attr('title', '...');
+    resultElement.attr('title', startFromId);
 
     $.get(
         ajaxUrl,
         function (data) {
+            resultElement.removeClass('fa-spinner fa-pulse');
+
             try {
-                resultData = JSON.parse(data);
+                let result = JSON.parse(data);
+                let validResult = result.result !== undefined;
+
+                if (validResult && result.result === 'OK') {
+
+                    if (result.resultData.lastId !== undefined) {
+                        processNextItem(firstOrLast, buildAjaxCallback, callerButton, url_base, result.resultData.lastId);
+                        return;
+                    }
 
-                if (resultData.result !== "undefined" && resultData.result === 'OK') {
-                    resultElement.removeClass('fa-spinner fa-pulse');
                     resultElement.closest('.list-group-item-action').find('.form-check-input').removeClass('to-process');
                     resultElement.addClass('fa-circle-check fa-regular text-success ' + itemType);
                     resultElement.attr('title', '');
                     processNextItem(firstOrLast, buildAjaxCallback, callerButton, url_base, 0);
                     return;
                 }
 
-                if (resultData.lastId !== "undefined") {
-                    processNextItem(firstOrLast, buildAjaxCallback, callerButton, url_base, resultData.lastId);
-                    return;
-                }
-
-                if (resultData.errors !== "undefined") {
-                    resultElement.removeClass('fa-spinner fa-pulse');
+                if (validResult && result.result === 'ERROR') {
```

Comment:
We have to display x if there is an invalid result also.
```suggestion
                if (!validResult || result.result === 'ERROR') {
```